src/modules/control/launch/control.launch

```diff
@@ -9,5 +9,6 @@
   
   <node pkg="rosserial_python" type="serial_node.py" name="serial_node">
     <param name="port" value="/dev/ttyACM0"/>
+    <param name="baud" value="115200"/>
```

Comment:
Is there any way to not have a duplicate parameter with [rc.launch's line 15](https://github.com/SJSU-AD/FusionAD/pull/117/files#diff-8bb9c6165b65db802daa34fb9c2f4994R12)? If we would like to test various baud rates in the future, it would be safer to only have one place to modify the baud rate.

Reply:
Redudancy also occurs on [line 8](https://github.com/SJSU-AD/FusionAD/pull/117/files#diff-263b7074c629aece3b3421ead3521b9bR8) and [line 13](https://github.com/SJSU-AD/FusionAD/pull/117/files#diff-263b7074c629aece3b3421ead3521b9bR13) of velocity_estimate.launch

Reply:
It does not seem like it, or at least I wouldn't know how to do it. It would require modifying the ros_serial.py package that comes with Arduino.

Reply:
In that case, can you document this somewhere easily accessible so others are aware of the duplication, and don't run into issues when attempting to test rc.launch, control.launch and velocity_estimate.launch independently in the future?

Reply:
Documentation relevant issue: #118